test-data/unit/check-python2.test

```diff
@@ -62,7 +68,6 @@ A.f('') # E: Argument 1 to "f" of "A" has incompatible type "str"; expected "int
 import typing
 raise BaseException, "a"
 raise BaseException, "a", None
-raise BaseException, "a", None, None # E: Exception must be derived from BaseException
```

Comment:
It's a SyntaxError now.  I removed it because 1) the presence of a syntax error potentially hides other errors that would take place later and 2) there's not much point in explicitly testing for syntax errors with the fast parser.